modules/k8s_eks_addons/gpu-operator.tf

```diff
@@ -0,0 +1,21 @@
+resource "helm_release" "gpu_operator" {
+  count = var.gpu_operator_config.enable ? 1 : 0
+
+  namespace         = "gpu-operator"
```

Comment:
Is it possible maybe to set the gpu-operator to kube-system namespace with all other addons?

Reply:
changed to kube-system namespace